site/source/user-manual/backups/backup-setup/backup-linux.rst

```diff
@@ -0,0 +1,126 @@
+
+        #. Check the box for "Allow others to create and delete files in this folder", then click "Create Share"
+
+            .. figure:: /_static/images/cifs/cifs-lin3.png
+                :width: 60%
+
+        #. Click "Add Permissions Automatically"
+
+            .. figure:: /_static/images/cifs/cifs-lin4.png
+                :width: 60%
+
+
+    .. group-tab:: Other Linux
+
+        #. Install Samba if it is not already installed.
+
+            * ``sudo pacman -S samba``                                      For Arch
+            * ``sudo apt install samba``                                    For Debian
+            * ``sudo yum install samba``                                    For CentOS/Redhat
+            * ``sudo dnf install samba``                                    For Fedora
+
+        #. Create a directory to share or choose an existing one and make note of its location (path).  For this example, it will be called ``test-backup`` and located at ``/home/user/test-backup``
+
+        #. Configure Samba, by adding the following to the end of the ``/etc/samba/smb.conf`` file:
+
+            .. code-block::
+
+                [test-share]
+                    path = /home/user/test-backup
+                    create mask = 0666
+                    directory mask = 0777
+                    read only = no
+                    guest ok = no
+
+            Where:
+
+            - ``[test-share]`` is the *Share Name*, or title of the entry, and can be called anything you'd like
+            - ``path`` should be the path to the directory you created earlier
+
+            Copy the remainder of the entry exactly as it is
+
+        #. Open a terminal and enter the following command, replacing ``YOUR-USER`` with your Linux username:
+
+                .. code-block:: bash
+
+                    sudo smbpasswd -a YOUR-USER
+
+                Create a password and keep it somewhere safe, such as Vaultwarden
+
+
+Connect Embassy
+---------------
+
+#. Go to *Embassy > Create Backup*.
+
+    .. figure:: /_static/images/config/embassy_backup.png
+        :width: 60%
+
+#. Click "Open".
+
+    .. figure:: /_static/images/config/embassy_backup0.png
+        :width: 60%
+
+#. Fill in the following fields:
+
+    * Hostname - This is the hostname of the machine that your shared folder is located on
+    * Path - This is the "Share Name" (name of the share in your samba config) and **not** the full directory path
+    * Username - This is the user on the remote machine that you used to create the shared directory
+    * Password - This is your user (from above) password
+
+    .. figure:: /_static/images/config/embassy_backup1.png
+        :width: 60%
+
+#. Click "Save".
+
+That's it!  You can now create encrypted, private "cloud" backups of all your Embassy data to your Windows machine!!
+
+.. tip:: Keeping a strong password and making regular backups are the humble responsibilities of digital independence, remain ever vigilant!
```

Comment:
If anything, this tip should be at the top of the page. Otherwise just remove it.